src/Razor/test/Microsoft.VisualStudio.LanguageServices.Razor.Test/ProjectSystem/DefaultWindowsRazorProjectHostTest.cs

```diff
@@ -749,8 +749,60 @@ public async Task OnProjectChanged_ReadsProperties_InitializesProject()
         Assert.Empty(_projectManager.GetProjects());
     }
 
+    [UITheory]
+    // Standard setup.  BaseIntermediateOutputPath ends in obj and IntermediateOutputPath starts with obj
+    [InlineData("C:\\my repo root\\solution folder\\projectFolder\\obj\\", "obj\\Debug\\net8.0", "C:\\my repo root\\solution folder\\projectFolder\\obj\\Debug\\net8.0")]
+    // ArtifactsPath in use as ../artifacts
+    [InlineData("C:\\my repo root\\solution folder\\projectFolder\\../artifacts\\obj\\projectName\\", "C:\\my repo root\\solution folder\\projectFolder\\../artifacts\\obj\\projectName\\debug", "C:\\my repo root\\solution folder\\artifacts\\obj\\projectName\\debug")]
```

Comment:
nit: C# verbatim strings (with the `@`) will get rid of all of those double slashes. Been there since C# 1.0 :smile: